src/templates/base/base.html

```diff
@@ -611,6 +616,16 @@
             }
           }
         });
+
+        var ebookLink = document.querySelector('#ebook-toc-download');
+        ebookLink && ebookLink.addEventListener("click", function (event) {
+          gtag('event', 'ebook-click', { 'event_category': 'clicks', 'event_label': 'toc-page', 'transport_type': 'beacon', 'value': 1 })
+        });
+
+        var ebookLink = document.querySelector('.ebook-menu-download');
```

Comment:
Could there be more than one element with this class, and would we need to add the event listener for each?

Reply:
Good point! There will be two: header and footer menu. Will fix when next on the computer.

Reply:
What about something generic to build on the `data-event` attribute? For example:

```js
var trackableElements = Array.from(document.querySelectorAll('[data-event]'));
trackableElements.forEach(function (trackableElement) {
  trackableElement.addEventListener('click', function (event) {
    gtag('event', event.target.data.event, { 'event_category': 'clicks', 'event_label': 'menu ' + document.location.pathname, 'transport_type': 'beacon', 'value': 1 });
  });
});
```

Reply:
OK all fixed. Needed a slightly different method for the `<select>` version but very similar.